tests/testthat/test-qenv_eval_code.R

```diff
@@ -110,8 +110,6 @@ testthat::test_that("an error when calling eval_code returns a qenv.error object
 })
 
 testthat::test_that("a warning when calling eval_code returns a qenv object which has warnings", {
-  pdf(nullfile())
-  on.exit(dev.off())
```

Comment:
I think these lines were here because of this issue

Reply:
okay , understood now